--- cheroot-6.5.5/cheroot.egg-info/requires.txt~	2019-04-30 06:17:40.000000000 +0000
+++ cheroot-6.5.5/cheroot.egg-info/requires.txt	2019-05-04 16:23:28.407037784 +0000
@@ -1,4 +1,3 @@
-backports.functools_lru_cache
 six>=1.11.0
 more_itertools>=2.6
 
--- cheroot-6.5.5/setup.cfg~	2019-04-30 06:17:41.000000000 +0000
+++ cheroot-6.5.5/setup.cfg	2019-05-04 16:23:55.019037414 +0000
@@ -57,7 +57,6 @@
 	setuptools_scm>=1.15.0
 	setuptools_scm_git_archive>=1.0
 install_requires = 
-	backports.functools_lru_cache
 	six>=1.11.0
 	more_itertools>=2.6
 
